pages/linux/losetup.md

```diff
@@ -0,0 +1,19 @@
+# losetup
+
+> Set up and control loop devices.
+
+- List loop devices with detailed info:
+
+`losetup -a`
+
+- Associate / Attach a device or file to a {{loop}} device:
+
+`sudo losetup /dev/{{loop}} /{{path/to/<file>}}`
+
+- Detach All {{loop}} devices:
+
+`sudo losetup -D`
+
+- Detach a specific {{loop}} device:
```

Comment:
We don't normally use the token syntax in the example descriptions. The word "specific" (or sometimes "given") provides sufficient context for connecting the tokens in the example with the corresponding parts of the description.